docs/hooks/hook_civicrm_alterUFFields.md

```diff
@@ -0,0 +1,37 @@
+# hook_civicrm_alterUFFIelds
+
+## Summary
+
+This hook allows for the modification of the available fields that are permissible for use within a profile.
```

Comment:
Can we just flesh this out a bit more

Reply:
I have fleshed it out a bit more now @eileenmcnaughton